README.md

```diff
@@ -258,6 +258,8 @@ To run the canopy server for production, please run:
 gunicorn canopy_server.app:app --worker-class uvicorn.workers.UvicornWorker --bind 0.0.0.0:PORT --workers WORKER_COUNT
 ```
 
+Note: replace `PORT` and `WORKER_COUNT` with values.
```

Comment:
No, no, that's actually a typo 🤦 .  
This was supposed to be `$PORT` and `$WORKER_COUNT`. And I guess these come from our docker image, where we explicitly set these env vars (with default, even if the user hasn't provided them).  
Can you fix the typo please?

Reply:
Actually, I'm not even sure why @miararoy has changed the main README to use these env vars - unlike in the docker case, if someone just `pip install canopy`, he is **not** guaranteed to have those env vars at all. I think we should just revert to what we had [before](https://github.com/pinecone-io/canopy/commit/44cfad11add34fc61d6e9f9f0ac4071ff24195bf).

@miararoy ?